.appveyor/appveyor_helper.bat

```diff
@@ -0,0 +1,40 @@
+@echo off
+setlocal enableDelayedExpansion
+
+rem for /f %%f in ('wmic cpu get NumberOfLogicalProcessors /value ^| find "="') do set %%f
+rem set /a "ncpus=%NumberOfLogicalProcessors%*%NUMBER_OF_PROCESSORS%"
```

Comment:
No this are not comment, actually while development I was trying to find out actual number for core available in system. Anyways this is not needed anymore I will remove them as we need only number of processors only.